<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DateTimeSetDialog</class>
 <widget class="QDialog" name="DateTimeSetDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>319</width>
    <height>208</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(236, 236, 236);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true">QFrame#frame{
	background-color:rgba(220, 220, 220, 204);
}

QFrame#label{
	background-color:rgba(0, 0, 0, 0);
}

QFrame#line{
	color:black;
}</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" rowstretch="2,1,4,2">
      <property name="leftMargin">
       <number>9</number>
      </property>
      <property name="topMargin">
       <number>9</number>
      </property>
      <property name="rightMargin">
       <number>9</number>
      </property>
      <property name="bottomMargin">
       <number>9</number>
      </property>
      <property name="horizontalSpacing">
       <number>9</number>
      </property>
      <property name="verticalSpacing">
       <number>6</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>16</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">font: bold 16pt &quot;Arial&quot;;
color: rgb(50, 50, 50);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QDialogButtonBox" name="buttonBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="standardButtons">
         <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
        </property>
        <property name="centerButtons">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QWidget" name="widget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QWidget#widget{
	border-radius: 2px;
	background-color: rgb(55, 55, 55);
}

QWidget QLabel{
	color: rgb(125, 125, 125);
	font: bold 24pt &quot;Arial&quot;;
	background-color: rgba(55, 55, 55, 0);
}

QWidget QSpinBox{
   color: rgb(125, 125, 125);
	font: bold 24pt &quot;Arial&quot;;
	background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:0.5, stop:0 rgba(220, 220, 220, 255), stop:0.957143 rgba(190, 190, 190, 255), stop:1 rgba(220, 220, 220, 255));
	selection-background-color: rgba(0, 0, 0, 0);
	selection-color: rgb(125, 125, 125);
	border: 0px solid;
	border-radius: 2px;
}
</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3" columnstretch="5,1,5,1,5">
         <property name="topMargin">
          <number>9</number>
         </property>
         <property name="horizontalSpacing">
          <number>0</number>
         </property>
         <item row="0" column="1">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QSpinBox" name="spinBox_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::NoButtons</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QSpinBox" name="spinBox_1">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::NoButtons</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QSpinBox" name="spinBox_3">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::NoButtons</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="Line" name="line">
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DateTimeSetDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>136</x>
     <y>94</y>
    </hint>
    <hint type="destinationlabel">
     <x>136</x>
     <y>52</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DateTimeSetDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>136</x>
     <y>94</y>
    </hint>
    <hint type="destinationlabel">
     <x>136</x>
     <y>52</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
